packages/integration-tests/src/test-project-setup/data_storage_auth_with_triggers.ts

```diff
@@ -236,6 +236,13 @@ class DataStorageAuthWithTriggerTestProject extends TestProjectBase {
       backendId,
       'AWS::IAM::Role'
     );
+
+    // ensure typed shim files are generated by checking for onDelete's typed shim
+    const typedShimStats = await fs.stat(
+      path.join(this.projectDirPath, '.amplify', 'function-env', 'onDelete.ts')
```

Comment:
Can we do this check on `defaultNodeFunction`? Where is the `onDelete` function coming from?

Reply:
We shouldn't test with `defaultNodeFunction` because we copy this shim from the source folder. So in cases where typed shim files are generating with another name (like is the case for this fix where shim files had `-lambda` at the end) E2E tests were still passing because the copied `defaultNodeFunction.ts` shim was never overwritten and still exists.

Reply:
Ah I see, that's a good catch